tests/test_coordinator.py

```diff
@@ -77,15 +84,15 @@ def test_coordinator_state_standby_round():
     # are connected
     coordinator = Coordinator(minimum_participants_in_round=1, fraction_of_participants=1.0)
 
-    assert coordinator.state == coordinator_pb2.STANDBY
```

Comment:
just a remark: I'm a little surprised that in the old version, `coordinator_pb2.STANDBY` seems to work just as well as `coordinator_pb2.State.STANDBY`...

Reply:
that's because the protobuf compiler basically generates two python thing out of an protobuf enum: 1) the actuall enum object and 2) a global constant for every enum value. calling it like `EnumName.EnumValue` is  just more readable that `EnumValue`.